### ISSUE TITLE
4.1.1 Braces are used where optional not implemented

### ISSUE THREAD

**Issue by galvanopus** [google/google-java-format#435]
Input:
```java
int a = 0;
if (2 < 3) a++;
System.out.println(a);
```

Expected:
```java
int a = 0;
if (2 < 3) {
  a++;
}
System.out.println(a);
```

Actual:
```java
int a = 0;
if (2 < 3) a++;
System.out.println(a);
```

**Comment by lucasbraune** [comment]
I am working on this issue.

**Comment by navaneeth-spotnana** [comment]
@lucasbraune That's great news.  The issue is there on java 11 too.

**Comment by anthonyvdotbe** [comment]
@galvanopus this can be closed as a duplicate of #51

**Comment by cushon** [comment]
#51
